frontend/src/actions/Home.js

```diff
@@ -0,0 +1,48 @@
+export const setLogin_email = (inputString) => {
+  return {
+    type: "SET_LOGIN_EMAIL",
+    payload: inputString
+  }
+}
+
+export const setLogin_password = (inputString) => {
+  return {
+    type: "SET_LOGIN_PASSWORD",
+    payload: inputString
+  }
+}
+
+export const setRegister_email = (inputString) => {
+  return {
+    type: "SET_REGISTER_EMAIL",
+    payload: inputString
+  }
+}
+
+export const setRegister_password = (inputString) => {
+  return {
+    type: "SET_REGISTER_PASSWORD",
+    payload: inputString
+  }
+}
+
+export const set_secondRegister_password = (inputString) => {
```

Comment:
picky: Can we use `setRegister_confirmedPassword` so that it'd be more consistent with other naming?

Reply:
sure